techAssignment/src/App.vue

```diff
@@ -1,78 +1,107 @@
 <script>
+import InputCommand from "./components/InputCommand.vue";
+import CommandHistory from "./components/CommandHistory.vue";
+
 export default {
+  name: "App",
+
   data() {
     return {
-      count: 0,
+      commandInput: "",
+      commandInputArray: [],
+      arrayValues: [],
+      keyValue: {},
     };
   },
+
+  methods: {
+    renderInput() {
+      var commandInputArray = this.commandInput.split(" ");
```

Comment:
should `commandInputArray` is a `const` instead of `var` em?
```suggestion
      const commandInputArray = this.commandInput.split(" ");
```